### ISSUE TITLE
Start dialog button

### ISSUE THREAD

**Issue by sahazu** [n3d1117/chatgpt-telegram-bot#253]
**[Feature idea]** Add an inline _"start dialog"_ button, before it is pressed, the messages will not go into the history and the bot will not remember them, so as not to waste extra tokens. When it is pressed, the bot will start to remember messages

**Comment by mustafa0x** [comment]
I agree. Conversations should be opt-in, perhaps with a separate slash command.
